src/simod/fuzzy_calendars/fuzzy_discovery.py

```diff
@@ -0,0 +1,182 @@
+import json
+from datetime import datetime, timedelta
+
+from bpdfr_discovery.log_parser import event_list_from_csv, discover_arrival_calendar, \
+    discover_arrival_time_distribution
+from bpdfr_simulation_engine.simulation_properties_parser import parse_simulation_model
+
+from simod.fuzzy_calendars.fuzzy_factory import FuzzyFactory
+from simod.fuzzy_calendars.intervals_frequency_calculator import ProcInfo, Method
+
+
+def build_fuzzy_calendars(csv_log_path, bpmn_path, json_path=None, i_size_minutes=15, angle=0.0, min_prob=0.1):
+    traces = event_list_from_csv(csv_log_path)
```

Comment:
When running your test script, this line fails for loan_MC_LU and loan_MC_HU with the following:

```
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 2007: invalid start byte
```

@orlenyslp Is it something that has to be fixed in `bpdfr_discovery.log_parser`?